src/api/users/emailVerification/sendVerificationEmail.ts

```diff
@@ -1,88 +1,104 @@
-import { RequestHandler, Request, Response, Express, query } from 'express'
-import { MongoClient, ObjectId, Timestamp } from 'mongodb'
+import { RequestHandler, Request, Response, Express, query } from "express";
```

Comment:
Is this your edit of a backend file?

Reply:
i didn't touch these file

Reply:
huh thats weird, you are probably behind in these files, don't worry about it we can just reject these changes